src/nvector/openmp/fmod/fnvector_openmp_mod.f90

```diff
@@ -762,20 +762,20 @@ function FN_VGetArrayPointer_OpenMP(v) &
 use, intrinsic :: ISO_C_BINDING
 real(C_DOUBLE), dimension(:), pointer :: swig_result
 type(N_Vector), target, intent(inout) :: v
-type(C_PTR) :: fresult 
-type(C_PTR) :: farg1 
+type(C_PTR) :: fresult
+type(C_PTR) :: farg1
 
 farg1 = c_loc(v)
 fresult = swigc_FN_VGetArrayPointer_OpenMP(farg1)
-call c_f_pointer(fresult, swig_result, [1])
+call c_f_pointer(fresult, swig_result, [FN_VGetLength_OpenMP(v)])
```

Comment:
`N_VGetLocalLength_OpenMP` is missing from the `.h` file

Reply:
This is using `FN_VGetLength_OpenMP`, as we discussed we do not need the local length function for the OpenMP vector.

Reply:
Misread the function name, this is good.